src/notebook.c

```diff
@@ -754,6 +779,314 @@ void notebook_remove_page(gint page_num)
 }
 
 
+/* Copied from sidebar.c.  We can place this in one place like utils.c. */
+static gboolean utils_filename_has_prefix(const gchar *str, const gchar *prefix)
+{
+	gchar *head = g_strndup(str, strlen(prefix));
+	gboolean ret = utils_filenamecmp(head, prefix) == 0;
+
+	g_free(head);
+	return ret;
+}
+
+
+/* Copied from sidebar.c.  We can place this in one place like utils.c,
+ * with a more formal naming convention. */
+static gchar *get_doc_folder(const gchar *path)
```

Comment:
I won't really use the word important there. But I simply find it convenient that the order would be the same, since traversing the documents using shortcut keys would be less confusing, whenever I'm looking at the sidebar.